### PR TITLE
Lost host address

### PR DESCRIPTION
**Pull request by xverges**
# Background
#### Link to issue 
N/A
#### Link to staging deployment URL 
Issue in the local development environment
#### Anything else the reviewers should know about the changes here

In our latest attempt to make life simpler for linux user, we broke the environment for MacOS users, overriding with an empty string `host.docker.internal`

# Changes
### Code changes

* Prevent the `host.docker.internal` override
* Wait for clouformation operations to complete before the next step

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR
Ugly Penguin

### CODE REVIEW

File: qc-runner/src/init.r [hunk 1]
```diff
@@ -27,7 +27,10 @@ load_config <- function(development_aws_server) {
 
     # running in linux needs the IP of the host to work. If it is set as an environment variable (by makefile) honor it instead of the
     # provided parameter
-    development_aws_server = Sys.getenv("HOST_IP", development_aws_server)
+    overriden_server = Sys.getenv("HOST_IP", "")
+    if (overriden_server != "") {
+        development_aws_server = overriden_server
+    }
```
Comment (alexvpickering) [inline qc-runner/src/init.r:33]: Why is this necessary? Should work fine:

image

Comment (xverges) [reply qc-runner/src/init.r:33]: The thing is that we don't have an unset `HOST_IP`. We have a `HOST_IP` set to an empty string and we ended up with `development_aws_server` set to an empty string.

Comment (alexvpickering) [reply qc-runner/src/init.r:33]: AHHH yes it's getting set in `app.js` even if the makefile doesn't:

``` javascript
....
const value = process.env[varNames[ii]] || '';
const replace = `__${varNames[ii]}__`;
```

tricksy ... good catch!

**Review by alexvpickering — APPROVED** [review]
Looks good to me. Thanks for correcting my blunder ...
